microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,85 @@
+| The language/region used for formatting date, time, numbers, etc. | The language in the **edge://settings/languages** list specified as the **displayed language** |
+
+### Regional Format
+To reflect a customer’s regional preferences (especially in the cases where the customer has changed their default preferences in the OS), Microsoft Edge is making a new Setting available in the “Language” section of Settings:
+
+![Share additional OS regional format with the option **limited** selected.](media/os-regional-setting-screenshot.msft.png)
+
+This access will allow websites to have visibility into more customer's specific regional preferences than was previously possible. Customers that changed their OS regional preferences may now see their preferences reflected automatically in more websites.
+
+## Sharing Setting
+Customers have three settings on Microsoft Edge to select the level of information their browser will share about their regions:
+
+* Never 
+* Limited (default) 
+* Always (not recommended). 
+
+#### Never Setting
+To minimize the risks, customers can disable the sharing of OS regional information by choosing **never**. This setting prevents websites from reading region information, but does block any specific OS regional customizations that the user has made from being exposed to website content. Instead, the browser'’'s display language is used to derive a default region. 
+
+#### Limited Setting (Default)
+The default recommended value is **limited**. With this setting, the above risks are mitigated by ensuring that “incompatible” combinations of language and region aren't revealed to websites. The “limited” value ensures that only valid combinations of the user's current browser display language preference and a given region value are used. **Limited** ensures that websites won't see a regional code anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The **limited** setting compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language in the message doesn't match, the feature acts as if **never** was selected (the regional information falls back to being derived from the browser’s display language). 
+
```

Comment:
global: If "Never", "Limited", etc are init-cap in UI, make them Init cap bold in .md file.